Runtime/LevelScripting/Logic/FlipFlopLogic.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+using NaughtyAttributes;
+
+namespace GameplayIngredients.Logic
+{
+    public class FlipFlopLogic : LogicBase
```

Comment:
Class Name seems to refer to electronics FlipFlop Twostate switch. But Callables lists are confusing because of their names:  `ConditionValid` and `ConditionInvalid`  (Valid is executed then Invalid... the next time?)
as condition is a private bool, we shan't assume that user will understand:

I'd choose another name for the two Callable[] members, such as `OnFlip`  and `OnFlop`

Otherwise, cool shortcut for a simplified state machine.

Reply:
Changed